### DISCUSSION TITLE
How do you add custom bottom, top, left, and right positions in tailwind.config.js?

### DISCUSSION THREAD

**Discussion by do-kevin** [tailwindlabs/tailwindcss#9896] [Help] [answered]
Hey there,

I'm trying to figure out how we can extend the utility classes of top, left, bottom, and right. So far, all I can find is the objectPosition and backgroundPosition in the default config file.

**Comment by wongjn** [comment, marked as answer]
As per [the documentation](https://tailwindcss.com/docs/top-right-bottom-left#customizing-your-theme), you can use the `spacing` or `inset` keys.
